### PR TITLE
Add interactive 3D component to Tizen 5.5

### PR DESCRIPTION
**Pull request by HunseopJeong**
- Add new guide pages for the interactive 3D component
- Add images for the interactive 3D preview

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

**Comment by mijin85cho** [comment]
Hello.
It looks like new Web API pages are added.
In that case, would you please update application/web/api/5.5/index.htm, also?

Also, please ask one of your team member to check and approve this PR.

**Comment by HunseopJeong** [comment]
@mijin85cho,
Okay, I will update the 'application/web/api/5.5/index.htm'.
I just want to check the staging page.
I will add the our team member after test it.

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm [hunk 1]
```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<link href="../styles.css" rel="StyleSheet" type="text/css">
+	<link href="../snippet.css" rel="StyleSheet" type="text/css">
+	<title>Coverflow</title>
```
Comment (DohyungLim) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: title need to modify Interactive3D.

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm [hunk 2]
```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<link href="../styles.css" rel="StyleSheet" type="text/css">
+	<link href="../snippet.css" rel="StyleSheet" type="text/css">
+	<title>Coverflow</title>
```
Comment (DohyungLim) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm]: ditto.

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+</head>
+<body onload="prettyPrint()" id="content">
+
+
+<h1>Interactive 3D</h1>
+
+<p>The interactive 3D component allows you to load a 3D model and controls the model freely.</p>
+
+<img src="../../images/wearable_interactive3D.png">
```
Comment (DohyungLim) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm]: please check image size.

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+</head>
+<body onload="prettyPrint()" id="content">
+
+
+<h1>Interactive 3D</h1>
+
+<p>The interactive 3D component allows you to load a 3D model and controls the model freely.</p>
+
+<img src="../../images/mobile_interactive3D.png" width="15%">
```
Comment (DohyungLim) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: please check image size.
and make sure your content is centered.

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+				<td class="option"></td>
+				<td class="option"></td>
+				<td class="option">Obrit controls allow the camera to orbit around a target.</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">autoplay</span></td>
+				<td class="option"></td>
+				<td class="option"></td>
+				<td class="option">Play animations of the model automatically.</td>
```
Comment (singa2000) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: Is there no default value of "autoplay"?
The "autoplay" has the "true, false" value ? or has the another value?

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">light</span></td>
+				<td class="option">string</td>
+				<td class="option">"0xffffff 0xffffff 1"</td>
+				<td class="option">A light source positioned directly above the scene(skyColor, groundColor, intensity).</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">controls</span></td>
+				<td class="option"></td>
```
Comment (singa2000) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: What's the input type of  "controls"?

**Comment by mijin85cho** [comment]
> @mijin85cho,
> Okay, I will update the 'application/web/api/5.5/index.htm'.
> I just want to check the staging page.
> I will add the our team member after test it.

Hello, @HunseopJeong
I think you can check the update on  https://docs.stg.tizen.org/staging/864.
Accurately
https://docs.stg.tizen.org/staging/864/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm and
https://docs.stg.tizen.org/staging/864/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm (hunk 1 above)
Comment (HunseopJeong) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: Done.

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm (hunk 2 above)
Comment (HunseopJeong) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm]: Done.

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm (hunk 3 above)
Comment (HunseopJeong) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm]: I used same size like as other wearable widgets.
Is it wroing?

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm (hunk 4 above)
Comment (HunseopJeong) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: I used the same size like as other mobile widgets.
Do we have to align it to the center?

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+<h2>Table of Contents</h2>
+
+<ol class="toc">
+
+		<li><a href="#default-selectors-interactive3D-mobile">Default Selectors</a></li>
+
+		<li><a href="#html-examples-interactive3D-mobile">HTML Examples</a></li>
+
+		<li><a href="#options-for-widget-interactive3D-mobile">Options</a></li>
+
```
[the remark this answers is not in the document]
Comment (HunseopJeong) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: Yes. It doesn't have the method.

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm (hunk 5 above)
Comment (HunseopJeong) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: Yes. There is no fault value. The execution of the attribute is determined by checking whether it is defined or not.

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm (hunk 6 above)
Comment (HunseopJeong) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: Same with "autoplay"

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+			<th>Default value</th>
+			<th>Description</th>
+		</tr>
+
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">src</span></td>
+				<td class="option">string</td>
+				<td class="option">""</td>
+				<td class="option">A string containing the path/URL of the file to be loaded.</td>
```
[the remark this answers is not in the document]
Comment (HunseopJeong) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm]: Okay. I will update it !

**Comment by DohyungLim** [comment]
@mijin85cho 
There seems to be no issue internally.
Can you check the grammar? If you find a problem, please enter a comment.

**Comment by mijin85cho** [comment]
Question to "Interactive 3D component allows you to load a 3D model and controls the model freely."
I hope you check whether isn't 
"Interactive 3D component allows you to load and control a 3D model freely." correct.

Same to below files:
 - mobile_extended_component_list.htm
 - wearable_extended_component_list.htm
 - mobile_Interactive3D.htm
 - wearable_interactive3D.htm

**Comment by HunseopJeong** [comment]
@mijin85cho, Done.

**Pull request merged**